play27-core/src/test/scala/play/api/test/ops/AsyncResultExtractorsSpec.scala

```diff
@@ -31,38 +40,72 @@ class AsyncResultExtractorsSpec extends AsyncFreeSpec with AsyncResultExtractors
       Ok(request.body)
     }
 
-    def echoJsonInHeader: EssentialAction = Action(parse.json) { request =>
+    def echoJsonInHeader: Action[JsValue] = Action(parse.json) { request =>
       val name = (request.body \ "name").as[String]
       val value = (request.body \ "value").as[String]
       Ok.withHeaders(name -> value)
     }
 
-    def echoJsonInCookie: EssentialAction = Action(parse.json) { request =>
+    def echoJsonInCookie: Action[JsValue] = Action(parse.json) { request =>
       val name = (request.body \ "name").as[String]
       val value = (request.body \ "value").as[String]
       Ok.withCookies(Cookie(name, value))
     }
 
-    def echoJsonInSession: EssentialAction = Action(parse.json) { request =>
+    def echoJsonInSession: Action[JsValue] = Action(parse.json) { request =>
       val sessionData = request.body.as[Map[String, String]]
       Ok.withSession(sessionData.toSeq: _*)
     }
 
-    def echoJsonInFlash: EssentialAction = Action(parse.json) { request =>
+    def echoJsonInFlash: Action[JsValue] = Action(parse.json) { request =>
       val flashData = request.body.as[Map[String, String]]
       Ok.flashing(flashData.toSeq: _*)
     }
 
-    def redirectToBody: EssentialAction = Action(parse.json) { request =>
+    def redirectToBody: Action[JsValue] = Action(parse.json) { request =>
       val url = (request.body \ "url").as[String]
       val status = (request.body \ "status").as[Int]
       Redirect(url, status)
     }
   }
 
-  def testSubject(methodName: String) = s"play23.AsyncResultExtrators.$methodName"
+  protected def method(name: String) = s"play26.AsyncResultExtractors.$name"
```

Comment:
play26?

Reply:
Yea... since this is open-source, I thought I should support Play 2.6, even though Rally doesn't support it.